content/english/supply chain analytics/1_Introduction.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Introduction to Supply Chain"
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 2
+---
+
+First thing's first... what is a supply chain? 
+
+{{% notice note %}}
+## What is a Supply Chain?
+
+“A supply chain moves a product or service from supplier to customer. It is the people, activities, information, and resources which move the product.” 
+
+{{% /notice %}}
+
+![Alt text](img/sandwich.png)
+
+Example: Your parents pack you a yummy sandwich to take to school for lunch the next day. Here’s what it takes to make that happen:
+1.	Farm grows the wheat
+2.	A facility turns that wheat into flour
+3.	A factory bakes that flour into bread
+4.	A delivery truck will take that bread from the factory and deliver it to the sandwich shop.
+5.	A chef makes the sandwich for you using that bread
+6.	You as the customer goes and buys that bread!
+
+
+This might sound simple. After all, when you’re craving Oreo cookies, all you have to do as the customer is go to the grocery store and pick up the Oreos from the shelf. You might not think of all the things that go on behind the scene - and there’s a lot of them. Here’s some examples of roles: 
```

Comment:
Let's not mention any brand names to not get into any trouble.

```suggestion
This might sound simple. After all, when you’re craving cookies, all you have to do as the customer is go to the grocery store and pick up the cookies from the shelf. You might not think of all the things that go on behind the scene - and there’s a lot of them. Here’s some examples of roles: 
```